### DISCUSSION TITLE
Loss Accumulation over several training steps (epochs)

### DISCUSSION THREAD

**Discussion by shaimove** [keras-team/keras-tuner#922] [Q&A]
Hi, 
I have limited GPU memory, and in order to increase my batch size, I want to perform loss accumulation over several training steps. 
How do I change the tuner object to do that? In a simple loop, it's very clear, but I using Bayesian Optimization for my project. 
thanks, 
Sharon

**Comment by NikitaShubin** [comment]
It's not related to Keras Tuner. 
KT using [model.fit](https://github.com/keras-team/keras-tuner/blob/e06bec6cac2172bd2cb038cca645578ef2ad307a/kerastuner/engine/tuner.py#L140):
IMHO, you shuld modify your model methods instead.
